### PR TITLE
docs(prometheus-scaler): 📝 Add customHeaders parameter

### PR DESCRIPTION
**Pull request by prashant-shahi**
Add custom headers parameter documentation for prometheus scaler.

### Checklist

- [X] Commits are signed with Developer Certificate of Origin (DCO)

Relates to https://github.com/kedacore/keda/issues/4206

Signed-off-by: Prashant Shahi <me@prashantshahi.dev>

### CODE REVIEW

File: content/docs/2.10/scalers/prometheus.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ triggers:
     # Optional fields:
     namespace: example-namespace  # for namespaced queries, eg. Thanos
     cortexOrgID: my-org # Optional. X-Scope-OrgID header for Cortex.
-    customHeaders: X-Client-Id=cid,X-Tenant-Id=tid,X-Auth-Token=authtoken # Optional. Custom headers to include in query.
+    customHeaders: X-Client-Id=cid,X-Tenant-Id=tid,X-Organization-Token=oid # Optional. Custom headers to include in query.
```
Comment (JorTurFer) [inline content/docs/2.10/scalers/prometheus.md:26]: Please, remove the token reference here

Comment (prashant-shahi) [reply content/docs/2.10/scalers/prometheus.md]: @JorTurFer updated

File: content/docs/2.10/scalers/prometheus.md [hunk 2]
```diff
@@ -36,6 +37,7 @@ triggers:
 - `activationThreshold` - Target value for activating the scaler. Learn more about activation [here](./../concepts/scaling-deployments.md#activating-and-scaling-thresholds).(Default: `0`, Optional, This value can be a float)
 - `namespace` - A namespace that should be used for namespaced queries. These are required by some highly available Prometheus setups, such as [Thanos](https://thanos.io). (Optional)
 - `cortexOrgID` - The `X-Scope-OrgID` header to query multi tenant [Cortex](https://cortexmetrics.io/) or [Mimir](https://grafana.com/oss/mimir/). (Optional)
+- `customHeaders` - Custom headers to include while querying the prometheus endpoint.
```
Comment (zroubalik) [inline content/docs/2.10/scalers/prometheus.md:40]: Might we worth noting here, that for auth related headers one should use Custom Authentication mentioned below.

Comment (prashant-shahi) [reply content/docs/2.10/scalers/prometheus.md]: @zroubalik updated

**Review by zroubalik — APPROVED** [review]

File: content/docs/2.10/scalers/prometheus.md [hunk 3]
```diff
@@ -22,7 +22,8 @@ triggers:
     activationThreshold: '5.5'
     # Optional fields:
     namespace: example-namespace  # for namespaced queries, eg. Thanos
-    cortexOrgID: my-org # Optional. X-Scope-OrgID header for Cortex.
+    cortexOrgID: my-org # Deprecated: Use custom headers instead to set X-Scope-OrgID header for Cortex. (see below)
```
Comment (tomkerkhove) [inline content/docs/2.10/scalers/prometheus.md:25]: Can you please align with this pattern here + in the list below?

`# DEPRECATED: This parameter is deprecated as of KEDA v2.10 in favor of {replacement} and will be removed in version 2.12`

**Review by tomkerkhove — CHANGES_REQUESTED** [review]
Deprecation announcement needs more work.

**Comment by prashant-shahi** [comment]
@tomkerkhove I have updated the deprecation messages as per the deprecation policy in KEDA governance.

**Review by zroubalik — APPROVED** [review]
thanks @prashant-shahi !

**Pull request merged**